docs/conf.py

```diff
@@ -13,6 +13,7 @@
 
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx.ext.autosectionlabel',
```

Comment:
In the updated doc strings, I added references to docs for other rules.
For example:
```
... see  the :ref:`quoted-strings` rule ...
```
This required that section labels be generated by the tooling.

However, since I am going to remove the `key-quoted-strings` rule, I will no longer need to reference other rule docs, so this change will not be needed.